sdk/core/core-compat/rollup.config.js

```diff
@@ -0,0 +1,3 @@
+import { makeConfig } from "@azure/dev-tool/shared-config/rollup";
```

Comment:
File may be removed if you include the changes from rollup -> dev-tool run bundle.